### DISCUSSION TITLE
Interpreting split_ci output

### DISCUSSION THREAD

**Discussion by BrauseT** [tpronk/splithalfr#23] [Q&A] [answered]
Hi!

I am calculating split half reliabilities using the splifthalfr-package. For consistency reasons I would like to report regular 95% confidence intervals, i.e. the parameter estimate and the corresponding upper and lower limits. In another discussion, I read that `split_ci` delegates the calculations to the function `bcajack` in the `bcaboot`-package. After reading [https://rdrr.io/cran/bcaboot/man/bcajack.html](https://rdrr.io/cran/bcaboot/man/bcajack.html) I figured that the standard limits are reported in the third column (std) of the `$lims`-table, whereas the parameter estimate can be found in the first row (est) and first column (theta) of the `$stats`-table (which is equal to the 0.5 value in the third column of the `$lims`-table). Is that correct?

Using the example output from the website above, this would lead to the following parameter estimate and 95% confidence interval: 
r = .51, 95% CI [.44, .57].

Example output (taken from [here](https://rdrr.io/cran/bcaboot/man/bcajack.html)): 

$lims
```
            bca      jacksd       std   pct
0.025 0.4349782 0.003100685 0.4421256 0.006
…
0.5   0.4989099 0.002384237 0.5065603 0.311
…
0.975 0.5615867 0.001996201 0.5709950 0.928
```

$stats
```
        theta       sdboot         z0            a     sdjack
est 0.5065603 0.0328754548 -0.2455895 -0.001290911 0.01864547
jsd 0.0000000 0.0007288154  0.0483231  0.000000000 0.00000000
```

**Comment by tayllana** [comment, marked as answer]
Yes, you are correct in your interpretation of the output from the `bcaboot` package. Here's how you can interpret it:

- The parameter estimate (theta) is indeed found in the first row (`est`) and first column (`theta`) of the `$stats` table. In your example, the parameter estimate (theta) is 0.5065603.
- The standard limits (std) are indeed reported in the third column (`std`) of the `$lims` table. These are essentially the confidence interval bounds. In your example, the lower limit would be 0.4421256 and the upper limit would be 0.5065603.

So, using the output you provided:

- The parameter estimate (r) is approximately 0.51 (rounded from 0.5065603).
- The 95% confidence interval for this estimate would be approximately [0.44, 0.57] (rounded from [0.4421256, 0.5065603]).

Your understanding seems accurate based on the information you've provided.

**Reply by tpronk** [reply]
Thanks for helping out @tayllana!

**Reply by BrauseT** [reply]
Just a quick note after quite some time: There was a typo / copy-paste error in your answer, @tayllana: 

> The standard limits (std) are indeed reported in the third column (std) of the $lims table. These are essentially the confidence interval bounds. In your example, the lower limit would be 0.4421256 and the upper limit would be **0.5065603**.

This value should have been **0.5709950**, I guess. Just pointing this out to avoid confusion. Thanks for the help again! :-)

**Comment by BrauseT** [comment]
Thanks a lot @tayllana and @tpronk! :-)

**Comment by johannesjuliusm** [comment]
I had been looking for the answer to the very same question everywhere in the package documentation and the tutorials but only found it here thanks to @tpronk. Thank you for clarifying @tayllana! @tpronk It would be so helpful if this were included in the tutorials. It could be as simple as one more line of code that extracts the lower and upper bound from the CI object or a comment explaining where the values are. Love this package and the flexibility it offers. It's helped me a lot understand the cognitive task data that I'm working with better. Thank you for maintaining this package!

**Reply by tpronk** [reply]
How about I add a link the [bcajack](https://rdrr.io/cran/bcaboot/man/bcajack.html) documentation in a future version, would that already do the trick?

**Reply by tpronk** [reply]
We might embark on a larger overhaul of bootstrapped confidence intervals, where we'll clear up a thing or two. See issue #26
